### PR TITLE
13047 Set up FEIS route, edit components, and list on project(s) page

### PR DESCRIPTION
**Pull request by CeciliaCY**
### Summary
13047 Set up FEIS route, edit components, and list on project(s) page

#### Task/Bug Number
[Fix AB 13047](https://dcp-paperless.visualstudio.com/ZAP%20Portals/_workitems/edit/13047)

### Technical Explanation
Same as Draft EAS

### CODE REVIEW

**Review by allthesignals — CHANGES_REQUESTED** [review]
Great! You are burning through this stuff, @CeciliaCY. 

Can you link up with @godfreyyeung and see what he's doing to fill out test coverage? I'd love to see an [acceptance test](https://guides.emberjs.com/release/testing/testing-application/) for this!

**Comment by CeciliaCY** [comment]
@allthesignals sure.

File: client/app/templates/project.hbs [hunk 1]
```diff
@@ -75,7 +75,7 @@
       </section>
     {{/if}}
 
-    {{#if @model.eisPackages.length}}
+    {{#if @model.technicalMemoPackages.length}}
```
Comment (godfreyyeung) [inline client/app/templates/project.hbs:78]: Thank you!

File: client/app/components/project/package-list-item.hbs [hunk 2]
```diff
@@ -187,12 +185,12 @@
       {{else}}
         <strong>
           {{#if (eq @package.dcpPackagetype (optionset 'package' 'dcpPackagetype' 'code' 'EIS'))}}
-            Filed
+            Final
```
Comment (godfreyyeung) [inline client/app/components/project/package-list-item.hbs:188]: Nice

**Review by godfreyyeung — APPROVED** [review]
Nice! Thank you Chengying!

**Pull request merged**